app/controllers/comments_controller.rb

```diff
@@ -362,12 +364,13 @@ def review_all
 
   def approve
     @comment.mark_as_ham!
-    redirect_to_all_comments(@comment.ultimate_parent, {show_comments: true})
+    redirect_to_all_comments(@comment.ultimate_parent, show_comments: true)
```

Comment:
Should we also have `authorize @comment` here? Only admins with certain roles should be able to mark comments as not spam. (Work creators should not be allowed to do this.)